README.md

```diff
@@ -24,7 +24,7 @@ The [Amazon FSx for OpenZFS](https://aws.amazon.com/fsx/openzfs/) Container Stor
 
 | Driver Version | [ECR Public](https://gallery.ecr.aws/fsx-csi-driver/aws-fsx-openzfs-csi-driver) Image |
 |----------------|---------------------------------------------------------------------------------------|
-| v0.1.0         | public.ecr.aws/fsx-csi-driver/aws-fsx-openzfs-csi-driver:v0.1.0                       |   
```

Comment:
Are we intentionally removing mention of v0.1.0 from the repo? Are you planning on doing this in future updates or is it just because it was a beta image?

Reply:
That was my initial thought but now I think it's best to leave it under past releases with a disclaimer that it was the beta version of the driver.